pkg/controller/service/config/OWNERS

```diff
@@ -1,14 +0,0 @@
-approvers:
-- api-approvers
```

Comment:
What about https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/service/config/v1alpha1/zz_generated. deepcopy.go? Either it should also be deleted or we should retain OWNERS.

Reply:
`https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/service/config/v1alpha1/zz_generated.deepcopy.go` is deleted together with OWNERS file in this PR.

Reply:
Sorry somehow missed it when I looked last time. Not sure how I missed it.